x-pack/plugins/security_solution/public/timelines/components/timeline/index.tsx

```diff
@@ -61,92 +56,18 @@ const StatefulTimelineComponent: React.FC<Props> = ({
   rowRenderers,
   timelineId,
 }) => {
-  const dispatch = useDispatch();
   const containerElement = useRef<HTMLDivElement | null>(null);
   const getTimeline = useMemo(() => timelineSelectors.getTimelineByIdSelector(), []);
-  const scopeIdSelector = useMemo(() => sourcererSelectors.scopeIdSelector(), []);
-  const {
-    selectedPatterns: selectedPatternsSourcerer,
-    selectedDataViewId: selectedDataViewIdSourcerer,
-  } = useDeepEqualSelector((state) => scopeIdSelector(state, SourcererScopeName.timeline));
-  const {
-    dataViewId: selectedDataViewIdTimeline,
-    indexNames: selectedPatternsTimeline,
-    graphEventId,
-    savedObjectId,
-    timelineType,
-    description,
-    sessionViewConfig,
-    initialized,
-  } = useDeepEqualSelector((state) =>
-    pick(
-      [
-        'indexNames',
-        'dataViewId',
-        'graphEventId',
-        'savedObjectId',
-        'timelineType',
-        'description',
-        'sessionViewConfig',
-        'initialized',
-      ],
-      getTimeline(state, timelineId) ?? timelineDefaults
-    )
+  const { graphEventId, timelineType, description, sessionViewConfig } = useDeepEqualSelector(
+    (state) =>
+      pick(
+        ['graphEventId', 'timelineType', 'description', 'sessionViewConfig'],
+        getTimeline(state, timelineId) ?? timelineDefaults
+      )
   );
 
   const { timelineFullScreen } = useTimelineFullScreen();
 
-  useEffect(() => {
```

Comment:
Timeline initialization logic moved to [use_initialize_timeline.ts](https://github.com/elastic/kibana/pull/170853/files#diff-053622e8e3c33e321ab93be487993fa03e8cee59a57a00e4243e83612ce6595e)